.circleci/build_autoinst.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+set -ex
+
+destdir=${1:-../os-autoinst}
+sha=${2}
+
+echo Building os-autoinst $destdir $sha
+git clone https://github.com/os-autoinst/os-autoinst.git "$destdir"
+( cd "$destdir"
+[ -z "$sha" ] || git checkout $sha
+autoreconf -f -i
+./configure
+make 
+)
```

Comment:
why do we need to build os-autoinst within openQA at all?

Reply:
because we might not have packages of the branch you depend on

Reply:
Yeah the idea for normal workflow: testing is always performed on the same "approved" list of dependencies including os-autoinst hash. And the bot will nightly detect changes in dependencies and submit corresponding PR, which will go trough CI and then get reviewed / merged. 
If somebody needs to try custom dependencies - they can edit `dependencies.txt` and/or e.g. change repositories in build_cache.sh - then CI will run tests against those custom dependencies.

Reply:
I don't understand how os-autoinst is any different to the other packages we have in devel:openQA:Leap

Reply:
As a start: os-autoinst is not in devel:openQA:Leap

Reply:
I see your point, but still if we used rpm for os-autoinst developments crossing front and backend would be plain impossible.

Reply:
In my understanding for fullstack tests we need development environment of os-autoinst in CI

Reply:
right. e.g. the Core.iso

Reply:
Yes, when I'm not mistaken we need the ISO from os-autoinst in openQA's fullstack test as well.